spring-kafka-test/src/main/java/org/springframework/kafka/test/utils/KafkaTestUtils.java

```diff
@@ -82,6 +84,16 @@ public static Map<String, Object> consumerProps(String group, String autoCommit,
 		return consumerProps(embeddedKafka.getBrokersAsString(), group, autoCommit);
 	}
 
+	/**
+	 * Set up test properties for an {@code <Integer, String>} consumer.
```

Comment:
I understand that it is a copy/paste, but what is that `@code` about?
Can we just take closer look to see what exactly Gary has meant with these JavaDocs?

Reply:
Thanks for your comments!
It's my bad. 
I fixed JavaDocs for `streamsProps()`, because it does not consider which Deserializer use to!
However, I think `public static Map<String, Object> consumerProps(String brokers, String group)` has a proper description, because it invokes `public static Map<String, Object> consumerProps(String brokers, String group, String autoCommit)`.

What do you think?

Reply:
Yeah... I see now. It talks about exactly what (de)serializers we configure over there for `Producer/Consumer` respectively.